.github/workflows/main.yml

```diff
@@ -29,20 +29,48 @@ jobs:
       - name: Nuke Build 🏗
         id: build
         shell: bash
-        run: ./build.sh --verbosity verbose
-        env:
-          OCTOVERSION_CurrentBranch:  ${{ github.head_ref || github.ref }} # For pull_request events we override the /refs/pull/xx/merge branch to the PR's head branch
-          OCTOVERSION_Patch : ${{ github.run_number }}
+        # Pass branch and patch number to Nuke OctoVersion 
+        # (for pull_request events we override the /refs/pull/xx/merge branch to the PR's head branch)
+        run: |
+          ./build.sh \
+            --OctoVersionBranch ${{ github.head_ref || github.ref }} \
+            --OctoVersionPatch ${{ github.run_number }}
+
+      - name: Tag release (when not pre-release) 🏷️
```

Comment:
tagging release (non pre releases) now we can access the version in the workflow yaml